library/src/main/java/com/bumptech/glide/request/target/ImageViewTarget.java

```diff
@@ -45,11 +45,11 @@ public void setDrawable(Drawable drawable) {
      */
     @Override
     public void onLoadStarted(Drawable placeholder) {
-        if(placeholder != null && view.getDrawable() != null){
-        view.setImageDrawable(placeholder);
+        if (placeholder == null && view.getDrawable() != null) {
```

Comment:
@sjudd if placeholder is null but imageview itself has drawable already, skip this method,thanks